components/roadmap-grid/RoadmapDetailedView.tsx

```diff
@@ -145,28 +147,38 @@ export function RoadmapDetailed({
   return (
     <>
       {isDevMode && <NumSlider msg="how many header ticks" value={numHeaderTicks} min={5} max={60} setValue={setNumHeaderTicks}/>}
-      {isDevMode && <NumSlider msg="how many grid columns" value={numGridCols} min={20} max={60} step={numHeaderTicks} setValue={setNumGridCols}/>}
-
-      <Box className={`${styles.timelineBox} ${ viewMode=='detail' ? styles.detailView : '' }`} >
-        <Grid ticksLength={numGridCols}>
-          {ticksHeader.map((tick, index) => (
-
-            <GridHeader key={index} tick={tick} index={index} numHeaderTicks={numHeaderTicks} numGridCols={numGridCols}/>
-          ))}
+      {isDevMode && <NumSlider msg="how many grid columns" value={numGridCols} min={20} max={60} step={numHeaderTicks} setValue={setNumGridCols} />}
+
+      {globalLoadingState.get() ? (
```

Comment:
> otherwise it breaks the console during testing.

What breaks exactly? I assume this is related to the issues with the asynchronously loaded children, and some of our state management being borked.

Reply:
a potentially simple fix for this (yet a step in the wrong direction imo) could be to go the provider route and simply handle things via prop-drilling